client/landing/gutenboarding/components/confirm-purchase-modal/style.scss

```diff
@@ -0,0 +1,37 @@
+.confirm-purchase-modal__content {
+	padding: 32px;
+	max-width: 560px;
```

Comment:
We'll probably want some consistency on modal widths across the flow, and should hence use variable or have this at component level.

No changes needed for now, just something to keep in mind.